xml/System.Text.Json.Serialization/JsonStringEnumConverter.xml

```diff
@@ -84,9 +84,9 @@ Reading is case insensitive. Writing can be customized by using a <xref:System.T
         <Parameter Name="typeToConvert" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeToConvert">The type to be checked.</param>
+        <summary>Determines whether the specified type can be converted to an enum.</summary>
+        <returns><see langword="true" /> if the type can be converted; otherwise, <see langword="false" />.</returns>
```

Comment:
Similarly here (from https://github.com/dotnet/dotnet-api-docs/pull/2889#discussion_r309537872):
<returns><see langword="true" /> if the type can be converted; otherwise, <see langword="false" />.</returns>

cc @Jozkee